index.html

```diff
@@ -1 +1,2 @@
-<div>Hello world</div>
\ No newline at end of file
+<div>Hello world</div>
+<button>foo</button>
```

Comment:
Hey @thangtheregista I assume you have wrong button label, let's change it from `foo` to `bar`.